### PR TITLE
feat: added input channel set up for messages

### PR DESCRIPTION
**Pull request by DmitryMarkov**
Added optional inputChannel to message payload

### CODE REVIEW

File: src/rest/methods/conversation.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export interface IMessage {
   }
   readonly createdAt: string
   readonly id: string
+  readonly inputChannel?: string
```
Comment (svlapin) [inline src/rest/methods/conversation.ts]: Does that have a different meaning (or possible values) than `EnumCommunicationMethodType` that was introduced recently? https://github.com/allthings/node-sdk/pull/310/files#diff-4b52d2a01def2cf9ca7bc9cd2e59d608R139

Comment (DmitryMarkov) [reply src/rest/methods/conversation.ts]: We had a conversation about it with Brand recently https://github.com/allthings/email-to-ticket/pull/93#discussion_r374752279
Also it should be the same as `EInputChannel` cockpit: https://github.com/allthings/cockpit/pull/2099/files#diff-9c8d980aa3afbc26a7392b2ef9e291d6R459-R464
Or we should extend `EnumCommunicationMethodType` with all possible channels?

Comment (svlapin) [reply src/rest/methods/conversation.ts]: > Also it should be the same as EInputChannel cockpit

Then I guess it should also be an enum here. Extending the type does not introduce a breaking change, while narrowing would do (e.g. `string` -> `enum`)

Comment (svlapin) [reply src/rest/methods/conversation.ts]: But it looks like it should be a separate from `EnumCommunicationMethodType` since not really related

Comment (brenthoneybone) [reply src/rest/methods/conversation.ts]: Correct, it's different from `EnumCommunicationMethodType` which is a communication type e.g. email, phone, what's app... Whereas EInputChannel is where the things came from e.g. cockpit, app, email, some external service etc

**Pull request merged**